File: app/src/main/java/ru/artempugachev/popularmovies/data/Review.java
package ru.artempugachev.popularmovies.data;

import com.google.gson.annotations.SerializedName;

/**
 * Data class for reviews
 */

public class Review {
    @SerializedName("id")
    private String id;

    @SerializedName("author")
    private String author;

    @SerializedName("content")
    private String content;

    @SerializedName("url")
    private String url;

    public String getAuthor() {
        return author;
    }

    public String getContent() {
        return content;
    }
}

File: app/src/main/java/ru/artempugachev/popularmovies/moviedetails/api/Review.kt
package ru.artempugachev.popularmovies.moviedetails.api

import com.google.gson.annotations.SerializedName

/**
 * Data class for reviews
 */

class Review (
    @SerializedName("id")
    val id: String,

    @SerializedName("author")
    val author: String,

    @SerializedName("content")
    val content: String,

    @SerializedName("url")
    val url: String
)
